apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/Components/UseExcluded.tsx

```diff
@@ -1,23 +1,119 @@
 import { CheckBox, CheckBoxOutlineBlank } from "@mui/icons-material";
-import { Box, Button } from "@mui/material";
-import { useContext, useMemo } from "react";
+import AddIcon from '@mui/icons-material/Add';
+import SettingsIcon from '@mui/icons-material/Settings';
+import { Box, Button, ButtonGroup, CardContent, Divider, Grid, Skeleton, Typography } from "@mui/material";
+import { Suspense, useCallback, useContext, useEffect, useMemo, useReducer } from "react";
 import { Trans, useTranslation } from "react-i18next";
+import ArtifactCardNano from "../../../../../Components/Artifact/ArtifactCardNano";
+import ArtifactFilterDisplay from "../../../../../Components/Artifact/ArtifactFilterDisplay";
+import CardDark from "../../../../../Components/Card/CardDark";
+import CardLight from "../../../../../Components/Card/CardLight";
+import CloseButton from "../../../../../Components/CloseButton";
+import ModalWrapper from "../../../../../Components/ModalWrapper";
 import SqBadge from "../../../../../Components/SqBadge";
 import { CharacterContext } from "../../../../../Context/CharacterContext";
 import { DatabaseContext } from "../../../../../Database/Database";
+import ArtifactCard from "../../../../../PageArtifact/ArtifactCard";
+import { artifactFilterConfigs, initialFilterOption } from "../../../../../PageArtifact/ArtifactSort";
+import useBoolState from "../../../../../ReactHooks/useBoolState";
+import useForceUpdate from "../../../../../ReactHooks/useForceUpdate";
+import useMediaQueryUp from "../../../../../ReactHooks/useMediaQueryUp";
+import { filterFunction } from "../../../../../Util/SortByFilters";
 import useBuildSetting from "../useBuildSetting";
 
-export default function UseExcluded({ disabled = false, artsDirty }: { disabled?: boolean, artsDirty: object }) {
+export default function UseExcluded({ disabled = false }: { disabled?: boolean }) {
```

Comment:
While we are here, can we rename this so it doesn't sound like a react hook?

Reply:
(nit-back) react hooks are `use...` and this is `Use...`